File: docs/snippets/all/archetypes/view_coordinates2d_simple.py
"""Use the math/plot convention for 2D (Y pointing up)."""

import rerun as rr

rr.init("rerun_example_view_coordinates2d", spawn=True)

rr.log("world", rr.ViewCoordinates2D.RU, static=True)  # Set Y-Up

rr.log(
    "world/points",
    rr.Points2D([(0, 0), (1, 1), (3, 2)]),
)

File: docs/snippets/all/archetypes/view_coordinates2d_simple.rs
//! Use the math/plot convention for 2D (Y pointing up).

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_view_coordinates2d").spawn()?;

    rec.log_static("world", &rerun::ViewCoordinates2D::RU())?; // Set Y-Up

    rec.log(
        "world/points",
        &rerun::Points2D::new([(0.0, 0.0), (1.0, 1.0), (3.0, 2.0)]),
    )?;

    Ok(())
}
